src/app/countriesCard/Card.tsx

```diff
@@ -1,35 +1,38 @@
 import Image from "next/image"
```

Comment:
This component should go to the / app / components folder.

Reply:
D'accord